conditional/__init__.py

```diff
@@ -63,6 +64,20 @@ def static_proxy(path):
 def default_route():
     return redirect('/dashboard')
 
+@app.errorhandler(404)
+@app.errorhandler(500)
+def route_errors(error):
+    username = request.headers.get('x-webauth-user')
+    member = ldap_get_member(username)
+    data = dict()
+    data['username'] = member.uid
+    data['name'] = member.cn
+    code = error.code
+    return render_template(request=request,
+                            template_name='404.html',
+                            error=str(error),
+                            error_code=code,
+                            **data), 404
```

Comment:
reference to error code in return is http://flask.pocoo.org/docs/0.12/api/#flask.Flask.errorhandler

also a small thing but if the template handles 404 and 500 should it be called `404.html`? maybe `error.html`